### DISCUSSION TITLE
I always run out of space on the C drive.

### DISCUSSION THREAD

**Discussion by ecorecipe** [Chia-Network/chia-blockchain#17382] [Q&A]
Hi

I have a question about the space on disk C during farming. 

The Chia & Fork app stop farming because the C: disk is full for no reason (My flot data is on the NAS)

But I will reboot, C drive space is recovered.

So, I reboot once a day.

I use 4T SSD.

Please help me.

**Comment by BrandtH22** [comment]
Hey @ecorecipe , by default the blockchain db is stored on the OS drive (the C: drive in this case).
If that does not have enough space then it is expected the client would crash and become nonfunctional. 

Also worth noting that running forks at the same time increases resource utilization and can cause issues while farming
